README.md

```diff
@@ -11,20 +11,52 @@ $ npm i judge-d -g
 ## Usage
 
 ```sh
-$ judge-d publish --help
+$ judge-d --help
 ```
 
 Help output:
 
+```
+Commands:
+  judge-d publish  Publish contracts
+  judge-d verify   Verify contracts
+
+Options:
+  --version  Show version number                                       [boolean]
+  --help     Show help                                                 [boolean]
+
+```
+
+## Commands
+
+```sh
+$ judge-d publish --help
+```
+
 ```
 Publish contracts
 
 Options:
-  --version         Show version number                                [boolean]
-  --help            Show help                                          [boolean]
   --url             Url to judge-d instance                  [string] [required]
   --pactsDir        Path to directory with pacts             [string] [required]
   --serviceName     Service name                             [string] [required]
   --serviceVersion  Service version                          [string] [required]
 
 ```
+
+```sh
+$ judge-d verify --help
+```
+
+```
+Verify contracts
+
+Options:
+  --url             Url to judge-d instance                  [string] [required]
+  --serviceName     Service name                             [string] [required]
+  --serviceVersion  Service version                          [string] [required]
+  --environment     Environment name                         [string] [required]
+  --outFile         HTML report filename                     [string] [required]
+  --outDir          Path to directory with generated report  [string] [required]
```

Comment:
Since the report is a single HTML file, having separate parameters for filename and directory is redundant. Just let the user specify the path to report.

Reply:
also, this parameter could be optional. If it is not passed, then no html report is generated